### DISCUSSION TITLE
Emails blocked > Emails forwarded?

### DISCUSSION THREAD

**Discussion by MrSuddenJoy** [anonaddy/anonaddy#699] [Q&A] [answered]
May I ask why its like this? Shouldnt be other way round? Please [see here](https://mega.nz/file/jFEkSDDZ#nP7kZXEPNvXdPX1v3MRofNDjBmHUYDVl53r3BMrF_mQ)

**Comment by willbrowningme** [comment, marked as answer]
If you have a lot of deleted aliases they will increase the blocked count total whenever someone attempts to send an email to them.

You can check your list of aliases with these sorting options - [https://app.addy.io/aliases?deleted=only&sort=-emails_blocked](https://app.addy.io/aliases?deleted=only&sort=-emails_blocked)
